static/src/javascripts/projects/commercial/modules/paid-containers.ts

```diff
@@ -0,0 +1,35 @@
+import bean from 'bean';
+import fastdom from 'fastdom';
+import { $$ } from '../../../lib/$$';
+
+const onKeyPress = (handler: EventListener) => (event: KeyboardEvent) => {
+	if (event.keyCode === 0x20 || event.keyCode === 0x0d) {
```

Comment:
[`keyCode`] is deprecated, and we should probably be using [`code`][] instead. At the very least, it’d be useful to know what these codes relate to. I have no idea what `0x20` or `0x0d` are 🤷‍♂️.

[`keyCode`]: https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode

[`code`]: https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/code

Reply:
It’s `Enter` and `Space` respectively. https://keycode.info/

`0x20` = `32` &rarr; `code`:`Enter`
`0x0d` = `13` &rarr; `code`:`Space`

Reply:
Interesting, Max. I had a look at the documentation and it looks like [`key`](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key) would also be a suitable (better?) alternative to [`code`](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/code) as it returns the value of the key being pressed (as opposed to the physical key on the keyboard).

Reply:
In this case, as they’re modifier keys, I’d be inclined to use `key`.

Reply:
Do you mean we should use `code` @mxdvl?

Reply:
`code` returns the behaviour of a physical keyboard and might not work on a virtual keyboard or an alternative input device, according to [Google](https://developers.google.com/web/updates/2016/04/keyboardevent-keys-codes). 
I agree that `e.key === " "` isn't very readable but it does seem more accessible for people using a virtual keyboard that doesn't mimic the layout of a standard keyboard (I think I'm entering extreme edge case territory here...).
TLDR: both work!

Reply:
I’m happy with either.